webapp/lib/main_ui.dart

```diff
@@ -465,6 +475,49 @@ class CodaUI {
     } // else, it's the last message, stop
   }
 
+  messageCoded(String messageID) {
```

Comment:
Consider `isMessageCoded` and please add a return type of bool to this method

Reply:
Changed, thanks